homeassistant/components/romy/binary_sensor.py

```diff
@@ -0,0 +1,104 @@
+"""Checking binary status values from your ROMY."""
+
+from dataclasses import dataclass
+
+from romy import RomyRobot
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import RomyVacuumCoordinator
+
+
+@dataclass(frozen=True)
+class RomyBinarySensorEntityDescription(BinarySensorEntityDescription):
+    """Immutable class for describing Romy data."""
+
+
+BINARY_SENSORS: list[RomyBinarySensorEntityDescription] = [
+    RomyBinarySensorEntityDescription(
+        key="dustbin",
+        translation_key="dustbin",
+    ),
+    RomyBinarySensorEntityDescription(
+        key="dock",
+        translation_key="dock",
+        device_class=BinarySensorDeviceClass.PRESENCE,
+    ),
+    RomyBinarySensorEntityDescription(
+        key="water_tank",
+        translation_key="water_tank",
+        device_class=BinarySensorDeviceClass.MOISTURE,
```

Comment:
tested it and it fits in that case:
{'dock': True, 'dustbin': True, 'water_tank': False, 'water_tank_empty': False}
if water tank_tank is False/Off it means water tank is no present, so robot will make a dry cleaning round, with water tank is present (True,On) it would make a wet cleaning round